includes/definitions/beagleboard.yaml

```diff
@@ -0,0 +1,31 @@
+os: beagleboard
+type: server
+group: unix
+text: BeagleBoard
+icon: beagleboard
+ifXmcbc: true
+ifname: true
+mib_dir:
+    - supermicro
```

Comment:
This doesn't seem right?

Reply:
I admit, just left this the same as in linux, as that's what this device defaults to without these changes (i.e. if OS not added, it falls back to linux). Remove these lines?

Thanks!

Reply:
Yes remove if not needed. I'm guessing beagleboard doesn't have its own mib

Reply:
Sorry, just to understand - you mean the full mib_dir section? I admit, not quite sure what is to be listed here, that's why I left it alone (i.e. as a copy from linux). Thanks!

Reply:
It's folders with mibs you want to include, to be able to use textual oids

Reply:
Got it, thanks! Removed.